File: weirdStringCase.js
toWeirdCase = (s) => {
    return s.split(' ').map(function (word) {
        return word.split('').map(function (letter, index) {
            return index % 2 == 0 ? letter.toUpperCase() : letter.toLowerCase()
        }).join('');
    }).join(' ');
}

console.log(toWeirdCase("Hello there"));

toWeirdCase = s => {
    a = s.split(' ')
    c = []
    d = []
    e = []
    for (i in a) {
        b = a[i].split('')
        for (x in b) {
            if (x % 2 == 0) {
                c.push(b[x].toUpperCase())
            } else {
                c.push(b[x].toLowerCase())
            }
        }

        d.push(c)
        e.push(d[0].join(''))
        c = []
        d = []
    }
    return e.join(' ')
}

console.log(toWeirdCase('Hello THERE')); 

File: weird_string_case.py
def to_weird_case(s):
    a = s.split(' ')
    b = []

    for i in a:
        b.append(list(i)) 
    
    print(b)
    for i in b:
        for count, x in enumerate(i):
            if count % 2 == 0:
                i[count] = x.upper()
            else:
                i[count] = x.lower()


    c = []
    for i in b:
        c.append("".join(i))

    return " ".join(c)

print(to_weird_case('Hi there hoow are you'))